src/status_im/ui/screens/extensions/add/views.cljs

```diff
@@ -45,25 +39,25 @@
               (i18n/label :t/extensions-disclaimer)]]
             [cartouche {:header (i18n/label :t/identifier)}
              [react/text {:style styles/text}
-              (str (get-in data ['meta :name]))]]
+              (str (get-in data [:meta :name]))]]
```

Comment:
is it still `meta` symbol in extension? or you decided to change symbols on keywords in edn?

Reply:
In the parsed extension it is keywords. The extension syntax didn't change.